frontend/src/plusButton/PlusButton.css

```diff
@@ -0,0 +1,7 @@
+.button {
+  padding: 40px;
```

Comment:
To parametrize spacing globally, this padding can be moved into the .tsx by using `makeStyles()` and `theme.spacing(4)`

Reply:
For example https://github.com/hack4impact-mcgill/mu-map/pull/62#discussion_r551409471